Gemfile

```diff
@@ -28,7 +28,6 @@ gem "therubyracer"
 gem "turbolinks"
 gem "uglifier"
 gem "unicorn"
-gem "pg"
```

Comment:
Hey @relotnek 👋 - it looks like you removed the gem. I just meant to do a:

```ruby
group :pg do
  gem "pg"
end
```

and then re-run bundle install or remove the pg entries from `Gemfile.lock` (bundling would do that). Lastly, updating the `openshift.md` doc to demonstrate the use of `RAILS_ENV=pg`

Reply:
I'll re-run the bundle and get it to build, but I already have the openshift group in there. The openshift doc includes some instructions on using the RAILS_ENV=openshift parameter. I just forgot to remove it from the main set of deps